giskard_vision/landmark_detection/types.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from typing import Tuple
+
+import numpy as np
+
+from giskard_vision.core.types import (
+    IMAGE_TYPE,
+    META_TYPE,
+    PredictionResultBase,
+    TypesBase,
+)
+
+LANDMARK_LABEL_TYPE = np.ndarray
```

Comment:
Our previous use case has only one face per image, should we support multiple faces as well?

I think here `np.ndarray` is an array of landmarks with fixed sizes. Maybe let's add a comment for it in any cases (multiple faces/single face)

Reply:
As well as the description of the label, which are actually a list of landmarks.